### PR TITLE
release 2.0.0alpha1

### PR DESCRIPTION
**Pull request by mbjones**
This PR is for an initial `2.0.0-alpha1` release which prepares the vegbank2 repository for our normal release cycle. It contains the following:

- Updated project structure and layout (#31)
- Contributing guidelines (#31)
- License files (#31)
- Initial work on database migration to newer Postgres version (#29)
- Initial work on Flyway database migration scripts (#29)
- Initial poetry build infrastructure for python package builds.

### CODE REVIEW

**Comment by mbjones** [comment]
Ping: @RWShelton @regetz I'd like to get this merged as a starting point to provide a better public face to the github repo and a starting point for incremental releases. Waiting on review from one of you.

**Comment by RWShelton** [comment]
Looks good to me, I've been working with the flyway stuff for a while and it's working well. My only question - I know it's a small commit, but does my last one with the baseline helm chart need to be reviewed individually before it goes in with this pull request? More of a process question for my own knowledge going forward than anything particular about this one.

**Comment by mbjones** [comment]
@RWShelton thanks. for your recent commits, in general and typically, each new major feature would indeed be in its own branch named after the feature it is implementing, and you would create a PR to pull that into develop. This process is described in detail in the [CONTRIBUTING.md](https://github.com/NCEAS/vegbank2/blob/develop/CONTRIBUTING.md) file. I think your current commit is fine though as a starting point in an alpha release.

I just added in the `tests` structure for pytest, so this branch now includes that as well. I will add a github action to run the tests in a sec. Then we can merge.

**Comment by mbjones** [comment]
Added GH Action to run `poetry run pytest` which is now passing. Can definitely be improved (it emits some warnings about doing a `pip install`, but its a start.

File: README.md [hunk 1]
```diff
@@ -9,17 +18,19 @@ VegBank provides a common storage system and web portal for accessing:
 - Plant taxonomy data
 - Community data
 
+VegBank is an open source, community projects.  We [welcome contributions](./CONTRIBUTING.md) in many forms, including code, graphics, documentation, bug reports, testing, etc.  Use the [DataONE discussions](https://github.com/DataONEorg/dataone/discussions) to discuss these contributions with us.
```
Comment (regetz) [inline README.md:21]: Do we want to keep the DataONE discussions link or is there something more appropriate?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+made to increase value to the community.  We strive to
+incorporate code, documentation, and other useful contributions quickly and
+efficiently while maintaining a high-quality software product.
+
+## 📤 Pull Requests
+We use the pull-request model for contributions. See [GitHub's help on pull-requests](https://help.github.com/articles/about-pull-requests/).
+
+In short:
+
+- add an [issue](https://github.com/DataONEorg/REPONAME/issues) describing your planned changes, or add a comment to an existing issue;
```
Comment (regetz) [inline CONTRIBUTING.md:37]: I assume we want to link to https://github.com/NCEAS/vegbank2/issues

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+- [🔬 Testing](#testing)
+- [🎨 Code style](#code-style)
+- [📄 Contributor license agreement](#contributor-license-agreement)
+
+## ✨ Types of contributions
+
+We welcome all types of contributions, including bug fixes, feature enhancements,
+bug reports, documentation, graphics, and many others.  You might consider contributing by:
+
+- Report a bug or request a new feature in our [issue tracker](https://github.com/DataONEorg/REPONAME/issues)
```
Comment (regetz) [inline CONTRIBUTING.md:18]: I assume we want to link to https://github.com/NCEAS/vegbank2/issues

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+- Fix a bug and contribute the code with a Pull Request
+- Write or edit some documentation
+- Sharing helpful tips or FAQ-type answers to users or future contributors
+- Create screenshots or tutorials of features of MetacatUI
```
Comment (regetz) [inline CONTRIBUTING.md:22]: Delete this bullet, and probably the next one unless we create a VegBank discussion board

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+incorporate code, documentation, and other useful contributions quickly and
+efficiently while maintaining a high-quality software product.
+
+## 📤 Pull Requests
+We use the pull-request model for contributions. See [GitHub's help on pull-requests](https://help.github.com/articles/about-pull-requests/).
+
+In short:
+
+- add an [issue](https://github.com/DataONEorg/REPONAME/issues) describing your planned changes, or add a comment to an existing issue;
+- on GitHub, fork the [repository](https://github.com/DataONEorg/REPONAME)
```
Comment (regetz) [inline CONTRIBUTING.md:38]: https://github.com/NCEAS/vegbank2

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+- on your computer, clone your forked copy of the repository
+- base your work on the `develop` branch and commit your changes
+- push your branch to your forked repository, and submit a pull-request
+- our team will be notified of your Pull Request and will review your changes
+- our team may request changes before we will approve the Pull Request, or we will make them for you
+- once the code is reviewed, our team will merge in your changes to `develop` for the next planned release
+
+## 🚀 Development Workflow
+
+Development is managed through the git repository at https://github.com/DataONEorg/REPONAME.  The repository is organized into several branches, each with a specific purpose.
```
Comment (regetz) [inline CONTRIBUTING.md:48]: https://github.com/NCEAS/vegbank2

File: README.md [hunk 7]
```diff
@@ -1,6 +1,15 @@
...
 ![VegBank](http://vegbank.org/vegbank/images/vegbank_caps170x40outline.jpg)
 
+
+## VegBank
+
+- **Authors**: Last, First (ORCID); ...
+- **License**: [Apache 2](http://opensource.org/licenses/Apache-2.0)
+- [Package source code on GitHub](https://github.com/NCEAS/vegbank2)
+- [**Submit Bugs and feature requests**](https://github.com/NCEAS/vegbank2/issues)
+- Contact us: support@dataone.org
```
Comment (regetz) [inline README.md:11]: Can/should we use an apporpriate VegBank email list?

**Review by regetz — APPROVED** [review]
I left a bunch of small comments where I saw especially DataONE and/or boilerplate links/etc that probably need to change, either as part of this PR or with the next one. Still marking as Approved to unblock this merge.

File: README.md (hunk 1 above)
Comment (mbjones) [reply README.md:21]: That's a good question. I will leave this as is, but I opened a ticket (#34)  to update this and replace it with a vegbank-appropriate solution. The README is a template and needs a lot of work still.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mbjones) [reply CONTRIBUTING.md:37]: yes, fixed locally and I will push shortly.

File: CONTRIBUTING.md (hunk 3 above)
Comment (mbjones) [reply CONTRIBUTING.md:18]: yes, will push shortly

File: CONTRIBUTING.md (hunk 4 above)
Comment (mbjones) [reply CONTRIBUTING.md:22]: I generalized it rather than deleting it.

File: CONTRIBUTING.md (hunk 5 above)
Comment (mbjones) [reply CONTRIBUTING.md:38]: Done.

File: CONTRIBUTING.md (hunk 6 above)
Comment (mbjones) [reply CONTRIBUTING.md:48]: Done

File: README.md (hunk 7 above)
Comment (mbjones) [reply README.md:11]: Changed to help@vegbank.org for now.

**Pull request merged**